modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc

```diff
@@ -411,9 +439,16 @@ class NDB_Form_Data_Team_Helper extends NDB_Form
             $qparams['CI'] = $candidate;
         }
         //filter for sites
-        if (($site !=null)&&($site!='All Sites')) {
-            $where         .= " AND ca.CenterId = :CID";
-            $qparams['CID'] = $site;
+        if ($site!='All Sites') {
+            if ($site == 'All User Sites') {
+                $user           = User::singleton();
+                $siteID         = $user->getCenterIDs();
+                $where         .= " AND FIND_IN_SET(psc.CenterID, :CID)";
+                $qparams['CID'] = implode(',', $siteID);
+            } else {
+                $where         .= " AND s.CenterID = :CID";
```

Comment:
@driusan 

CenterID is coming now from the session table instead of the candidate table.